### DISCUSSION TITLE
Sign Out

### DISCUSSION THREAD

**Discussion by go4cas** [wavezync/vue3-google-signin#42] [Q&A]
Looking at the docs, I could not find a way for an authenticated user to log out. Is the the `idRevoke` helper the way to achieve this?

**Comment by kasvith** [comment]
it doesn't really store the user state. So once you obtained the tokens you need to store it somewhere you need(like in cookies etc)
When user signs out clear out tokens from your app state

idRevoke is used to revoke the ID Token. Read more [here](https://developers.google.com/identity/gsi/web/guides/revoke)
